changelog/upgrade-notes.md

```diff
@@ -31,6 +31,10 @@ but with additional css:
 }
 ```
 
+#### Removal of `ol.source.TileVector`
+
+The experimental `ol.source.TileVector` class has been removed. Its functionality is now provided by `ol.source.VectorTile`. The API is the same, except for the `tileLoadFunction`, which is no longer an `ol.TileVectorLoadFunctionType`, but an `ol.TileLoadFunctionType`.
+
```

Comment:
ol.source.TileVector was a vector source with some tile functionality on top. ol.source.VectorTile is a tile source with some vector functionality on top, and works with ol.layer.VectorTile. The removal of the former came quite late in the development process, when I decided that only 2 methods needed to be added to make ol.source.VectorTile work for the existing examples.

Do you think it would make sense to change the name back? And maybe also rename ol.layer.VectorTile to ol.layer.TileVector?